hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/distributions/dstable/DistributionSetTable.java

```diff
@@ -233,7 +192,13 @@ protected boolean isMaximized() {
 
     @Override
     protected List<TableColumn> getTableVisibleColumns() {
-        return HawkbitCommonUtil.getTableVisibleColumns(isMaximized(), false, i18n);
+        final List<TableColumn> columnList = super.getTableVisibleColumns();
+        if (isMaximized()) {
+            columnList.add(new TableColumn(SPUILabelDefinitions.VAR_VERSION, i18n.get("header.version"), 0.1f));
```

Comment:
What about extracting the float value to a local variable? 
So you'll write the long complex add-call only once.

`float value = isMaximized() ? 0.1f : 0.2f;`
`columnList.add(new TableColumn(SPUILabelDefinitions.VAR_VERSION, i18n.get("header.version"), value ));`

Maybe you'll can find a better name for "value"